### PR TITLE
Move settings to feature module

### PR DESCRIPTION
**Pull request by AdityaKumdale**
## Issue Fix

Jira Task: [MW-49](https://mifosforge.jira.com/browse/MW-49)

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

[MW-49]: https://mifosforge.jira.com/browse/MW-49?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: feature/settings/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+plugins {
+    alias(libs.plugins.mifospay.android.feature)
+    alias(libs.plugins.mifospay.android.library.compose)
+}
+
+android {
+    namespace = "org.mifospay.settings"
```
Comment (therajanmaurya) [inline feature/settings/build.gradle.kts]: rename `org.mifospay.settings` to `org.mifospay.feature.settings`

File: feature/settings/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+dependencies {
+    implementation(projects.core.data)
+    implementation(projects.feature.passcode)
+    implementation(projects.feature.auth)
+    implementation(projects.feature.editpassword)
+    implementation(libs.mifosPasscode)
+    implementation(libs.androidx.appcompat)
```
Comment (therajanmaurya) [inline feature/settings/build.gradle.kts:16]: why we need `org.mifospay.settings`

File: feature/settings/src/main/kotlin/org/mifospay/feature/settings/SettingsViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    private val blockUnblockCommandUseCase: BlockUnblockCommand
+) : ViewModel() {
+
+    fun logout() {
+        mLocalRepository.preferencesHelper.clear()
+    }
+
+    fun disableAccount() {
+        // keep it disabled for now
+        if (0 * 67 == 0) {
```
Comment (therajanmaurya) [inline feature/settings/src/main/kotlin/org/mifospay/feature/settings/SettingsViewModel.kt:24]: @AdityaKumdale create a ticket to handle from remote config

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by AdityaKumdale** [comment]
closed due to package errors

**Pull request closed without merging**